util/db.go

```diff
@@ -113,7 +114,10 @@ func InitDB() error {
 
 		user := new(model.User)
 		user.Username = defaultUsername
-		user.Password = defaultPassword
+		user.Password, err = bcrypt.GenerateFromPassword(defaultPassword, bcrypt.MaxCost)
```

Comment:
`GenerateFromPassword` function returns 2 values, the first value's type is `[]byte` which cannot be assigned to `user.Password` which has `string` type. Please correct it.